### ISSUE TITLE
"Buy 0 null at price 0 null/null" pop up

### ISSUE THREAD

**Issue by CryptoChaser13** [BuildOnViction/bug-reports#99]
Hello. I tried to make a buy order with the cheapest price on the orderbook and maximize my order to 100% so it won't be filled easier. It was submitted on open orders And now i cancelled my order then a confirmation pop-out came out. I tried to turn on 'Auto Confirm for dex.testnet.tomochain.com' and i cancelled my order. Everything is normal. 

Consequently, i tried to make another buy order, same cheapest price and 100% so it wont get filled immediately. Now, its submitted on open orders. As you can see on the picture below, it was already filled 3% and then i decided to cancel out again the order. This time it dont require confirmation anymore because i turned the auto confirm on with the cancellation of first transaction. Suddenly, a pop-out came and seems need to be fixed:
"Buy 0 null at price 0 null/null". 

![Screenshot_20200126-162932_Tomo Wallet](https://user-images.githubusercontent.com/46350890/73132756-1ff52e80-3fd4-11ea-8fec-b3cdb4c55cf8.jpg)

I hope i have figured out this bug. 

Here is my Tomo wallet address:
0xafb82d9c2609ead9534774b16f9d1377d2decea3

**Comment by thanhson1085** [comment]
Seems issues with TomoWallet on some Android devices. We can not reproduce it now. But I will keep the issue open and follow it up.

**Comment by CryptoChaser13** [comment]
Thank you sir. I'll just wait for an update.

**Comment by CryptoChaser13** [comment]
Hello sir @thanhson1085 I would just like to follow up this issue. Thank you.

**Comment by CryptoChaser13** [comment]
![IMG_20200208_195201_130](https://user-images.githubusercontent.com/46350890/74084705-657b1800-4a26-11ea-9b75-97ae79be574e.jpg)

I alao encountered this issue on different device , what i used on the first post was Samsung Galaxy A50, now i encountered issue on Huawei Nova 2i

**Comment by thanhson1085** [comment]
Thank you for the information.

**Comment by thanhson1085** [comment]
Issue with TomoWallet Android version.

**Comment by CryptoChaser13** [comment]
Yes sir. Haven't tried it on IOS, but encountered issue this issue of Dex on app in any android.

**Comment by thanhson1085** [comment]
I meant that It is a TomoWallet MObile bug, not TomoDex bug.
Thank you for your report, I will forward to TomoWallet Android team.

**Comment by CryptoChaser13** [comment]
So will it be rewarded sir?

**Comment by CryptoChaser13** [comment]
I encountered it on dexchange at android app.

**Comment by CryptoChaser13** [comment]
What bug level here sir? @thanhson1085
Here is my Tomo wallet address:
0xafb82d9c2609ead9534774b16f9d1377d2decea3

**Comment by thanhson1085** [comment]
We only reward for TomoDEX, TomoRelayer bug report. This is tomowallet's bug.

**Comment by CryptoChaser13** [comment]
But it falls on tomodex sir. At https://dex.testnet.tomochain.com running on dapps. Its also indicated in the code.

**Comment by CryptoChaser13** [comment]
![IMG_20200213_132018_739](https://user-images.githubusercontent.com/46350890/74403834-7d171f80-4ddd-11ea-9284-6acaf79edd86.jpg)

**Comment by thanhson1085** [comment]
TomoDEX is a web application run on TomoWallet's DApp browser. 
If there is an bug with TomoWallet, I may affect to TomoDEX.

**Comment by CryptoChaser13** [comment]
So, no reward for this sir? Hmm.

**Comment by thanhson1085** [comment]
Yes. No reward. It is bug of TomoWallet.

**Comment by CryptoChaser13** [comment]
Oh. I thought it has reward, cause i also ask it on tomo tehnical group on telegram.

**Comment by CryptoChaser13** [comment]
Thank you.
